modules/log6x-collection-setup.adoc

```diff
@@ -92,7 +92,7 @@ rules:                                              <1>
       - logs                                        <7>
     verbs:                                          <8>
       - create                                      <9>
-Annotations
+----
```

Comment:
Good catch. I guess the core OCP bugs will carry a bug :scream:  

![Screenshot From 2025-02-18 15-01-43](https://github.com/user-attachments/assets/5418ae28-e1c9-4b59-93fc-06d3dbda2648)

Reply:
For a few days yes, unfortunately! But it's been there for some time so a few more days shouldn't hurt much i guess.